style.css

```diff
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  background: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  background-color: white;
+  z-index: 1000;
+  top: 0;
+  margin: auto;
+  align-self: center;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  top: 30px;
+}
+
+nav img {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  right: 2vh;
+  top: 2vh;
+  border-radius: 8px;
+}
+
+#sec1 {
+  position: relative;
+  background: white;
+  border-bottom-left-radius: 20%;
+  padding-top: 180px;
+  height: 100vh;
+  margin: auto;
+  background-image: url("./assets/bg.svg");
+  background-repeat: no-repeat;
+  background-position: center top;
+  background-size: 100%;
+  background-position-y: 5vh;
+}
+
+#sec1 div {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  top: -18vh;
+}
+
+#logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 0.9em;
+  line-height: 20px;
+  color: #6070ff;
+  padding: 10px 12px;
+  position: absolute;
+  width: 300px;
+  height: 40px;
+  left: 2vh;
+  top: 2.5vh;
+}
+
+/* Section */
+#nm {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.5em;
+  line-height: 18px;
+  color: #344563;
+  height: 150px;
+  width: 100%;
+}
+
+.desc {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 50 !important;
+  font-size: 0.8em;
+  line-height: 1.2rem;
+  color: #344563;
+  height: 146px;
+  width: 100%;
+  position: relative;
+  top: -12vh;
+}
+
+.lnk {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.8rem;
+  line-height: 24px;
+  color: #7f8cff;
```

Comment:
- [ ]  we should try to stick to the Figma design as much as possible as recommended by the project requirements. I would suggest you to use details of font sizes, margins, and paddings from Figma design for every element in your project so that it would be pixel-perfect.
your design 
![saw](https://user-images.githubusercontent.com/96683843/211891018-e3bb3c3b-ff10-4941-ac05-eb1b793f232c.jpg)

Figma design
![sp1](https://user-images.githubusercontent.com/96683843/211891073-c98d3e04-76b9-4eba-a502-d7caec38644a.jpg)

Reply:
It looks fine on my end 
![proof](https://user-images.githubusercontent.com/114868555/211893265-7dee0797-ffcd-4ae9-a90e-c8ffb053aa73.JPG)
But i'll see what i can do about it :)

@Zuhaib042